<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ML Lab  4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="exec1_btn">
        <property name="text">
         <string>Execute Task #1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec2_btn">
        <property name="text">
         <string>Execute Task #5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec3_btn">
        <property name="text">
         <string>Execute Task #7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec4_btn">
        <property name="text">
         <string>Execute Task #8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec5_btn">
        <property name="text">
         <string>Execute Task #9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec6_btn">
        <property name="text">
         <string>Execute Task #13</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="output">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
